src/include/btree.h

```diff
@@ -64,6 +64,15 @@
  */
 #define WT_BTREE_MIN_SPLIT_PCT 50
 
+/*
+ * Normalized position constants
+ */
+#define WT_NPOS_MID 0.5           /* Middle of the current page */
+#define WT_NPOS_LEFT -1e-8        /* Leftmost position in the current page or previous page */
+#define WT_NPOS_RIGHT (1. + 1e-8) /* Rightmost position in the current page or next page */
+#define WT_NPOS_INVALID -1.0      /* Store this as an invalid position */
+#define WT_NPOS_IS_INVALID(pos) ((pos) < 0.0)
```

Comment:
I raised this with Yury on slack while trying to understand npos in general but I think INVALID being <0 and WT_NPOS_LEFT also being less than zero means that left is also invalid. 

i.e. WT_NPOS_IS_INVALID(WT_NPOS_LEFT) = true. Which seems odd to me.

Reply:
Yury's response which makes sense to me: 
```
WT_NPOS_LEFT is something that you pass to __wt_page_npos
WT_NPOS_IS_INVALID you apply to the result of __wt_page_npos which is always between 0 and 1
```

Reply:
I've updated comment explaining this. Please see if they are clear enough or I need to do something better.